AiProgrammerApp.ts

```diff
@@ -14,10 +14,28 @@ import { settings } from './settings/settings';
 import { IUIKitResponse, UIKitActionButtonInteractionContext, UIKitBlockInteractionContext, UIKitViewCloseInteractionContext, UIKitViewSubmitInteractionContext } from '@rocket.chat/apps-engine/definition/uikit';
 import { ElementBuilder } from "./lib/ElementBuilder";
 import { BlockBuilder } from "./lib/BlockBuilder";
-import { ExecuteActionButtonHandler } from './handlers/ExecuteActionButtonHandler';
 import { ExecuteBlockActionHandler } from './handlers/ExecuteBlockActionHandler';
 import { ExecuteViewClosedHandler } from './handlers/ExecuteViewClosedHandler';
 import { ExecuteViewSubmitHandler } from './handlers/ExecuteViewSubmitHandler';
+import { IUser } from "@rocket.chat/apps-engine/definition/users";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+import { ProcessorsEnum } from "./enum/Processors";
+import {
+    IAuthData,
+    IOAuth2Client,
+    IOAuth2ClientOptions,
+} from "@rocket.chat/apps-engine/definition/oauth2/IOAuth2";
+import { createOAuth2Client } from "@rocket.chat/apps-engine/definition/oauth2/OAuth2";
+import { sendNotification, sendMessage, sendDirectMessage } from "./helpers/message";
+import { clearInteractionRoomData, getInteractionRoomData } from "./persistance/roomInteraction";
+import { deleteOathToken } from "./processors/deleteOAthToken";
+import { SendReminder } from "./handlers/SendReminder";
```

Comment:
Why do you have this sendReminder Function in here? 
its specific to github app's PR review reminder feature

Reply:
deleted this unnecessary function in newest commit